_posts/2015-11-11-full-stack-es6-with-webpack-and-babel.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "On the way to full stack ECMAScript 6"
+date: "2015-11-16 12:00:00"
+icon: code
+categories: tech-stories
+authors: ["Paolo"]
+---
+
+**JavaScript is moving fast these days.** With all the languages out there compiling to it, JS itself is making a big leap forwards with the standardisation of its next version called _ECMAScript 6_ (or _ES6_, or _ECMAScript 2015_, or _ES2015_. [Naming is probably the hardest programming related task](https://twitter.com/a_ramella/status/656522461480099840), and if you think that _ECMAScript sounds like a skin disease_, you're [in good company with the inventor of JavaScript himself](https://mail.mozilla.org/pipermail/es-discuss/2006-October/000133.html)).
+
+In this post, I want to share things we've learned during our ongoing adoption of the new language features.
+
+## JS @epages
+My team and I are working on an upcoming part of the ePages software that provides an easy to use drag-and-drop editor for the layout and appearance of an ePages shop, and serves the actual shop web site by displaying all product and editorial data in that layout.
```

Comment:
I think the sentence is perhaps too long what do you think about breaking it up like this:

My team and I are working on an upcoming part of the ePages software that provides an easy to use drag-and-drop editor for the layout and appearance of an ePages shop. It serves the actual shop website by displaying all product and editorial data in that layout.